README.md

```diff
@@ -0,0 +1,96 @@
+# Complete integration sample for the Corbado web component
+This is a sample implementation of frontend and backend where the Corbado web component is integrated.
+
+## 1. File structure
+    .
+    ├── ...
+    ├── config                        
+    │   └── routes.yaml                 # Assigns paths to controller methods    
+    ├── src                             
+    │   ├── Controller                  
+    │   │   ├── BackendController.php   # Manages endpoints for backend
+    │   └── └── FrontendController.php  # Manages endpoints for frontend
+    ├── templates                     
+    │   ├── home.html.twig              # Home page which you only get to see if you are logged in
+    │   └── login.html.twig             # Login page which contains the Corbado web component
```

Comment:
Gibts noch so eine Startseite bzw. richtige Homepage am Anfang oder landet man direkt auf login.html.twig?

Reply:
Wenn man nicht angemeldet is wird man zu /login weitergeleitet

Reply:
bitte dann auch hinschreiben, dass es zeitgleich die Startseite ist